runtime/compiler/z/codegen/S390PrivateLinkage.cpp

```diff
@@ -1230,7 +1230,7 @@ J9::Z::PrivateLinkage::createPrologue(TR::Instruction * cursor)
       cursor = generateS390LabelInstruction(cg(), TR::InstOpCode::label, firstNode, reStartLabel, cursor);
       }
 
-      // End of stack overflow checking code ////////////////////////
+      // End of stack overflow checking code////////////////////////
```

Comment:
It looks like you made an inadvertent change on this line. May I ask you to add back the space that you removed from this line?

```
      // End of stack overflow checking code ////////////////////////
```